app/views/trips/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<div id="app">
+
+</div>
```

Comment:
if you are routing your react pages to `homes#index` (in routes.rb), then you do not need these erb views. All react pages should be directed to the `homes#index` which has the
```
<div id="app">
</div>
```
and can be reused by multiple react components